### PR TITLE
DHP-92 Navigation to consent selection at Fitbit

### PR DESCRIPTION
**Pull request by ncbui**
## Description of change

Adds a way for Veterans to consent to share their Fitbit data with the VA. 

Adds new endpoint `/dhp_connected_devices/fitbit`, `/dhp_connected_devices/apidocs` to be called from the frontend, along with apidocs and tests. Also adds new client to connect to fitbit. 

## Original issue(s)
department-of-veterans-affairs/va.gov-team##9448 

## Things to know about this PR
* There are additions to a `settings.yml` file for the `dhp` module. They will vary by environment, we've only set dev environment for now. 
* There a feature flag,  `dhp_connected_devices_fitbit`
* Is there some Sentry logging that was added? No
* Are there any Prometheus metrics being collected? What Grafana dashboard were they added do? No
* Are there Swagger docs that were updated? Docs added for `/dhp_connected_devices/`
* Is there any PII concerns or questions? No

Integration tests were added for the controller and unit tests added for the fitbit client.

### CODE REVIEW

**Comment by thilton-oddball** [comment]
@yonidoronpeters It looks like your team is in the CODEOWNERS file for your module directory, so you should be able to self-review any future PRs, as long as all of the changes you are making are completely contained within your modules directory. This PR wouldn't qualify because you have file changes that fall outside of your module.

**Comment by yonidoronpeters** [comment]
@thilton-oddball thank you for the quick response! we did add our team as a codeonwer for this module, however we get this error/warning due to the DHP team not being added to the vets-api repo
Screen Shot 2022-04-04 at 11 44 17 AM
which also has the effect of not letting me add the DHP team as a reviewer to this PR

**Comment by thilton-oddball** [comment]
Ok, the DHP team is added to the repo now.

File: modules/dhp_connected_devices/spec/lib/fitbit_client_spec.rb [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe DhpConnectedDevices::Fitbit::Client do
```
Comment (thilton-oddball) [inline modules/dhp_connected_devices/spec/lib/fitbit_client_spec.rb:5]: I'm getting a failure on line 5 when I run this spec locally.

NameError:
  uninitialized constant DhpConnectedDevices::Fitbit
  Did you mean?  FitbitAPI
  ./modules/dhp_connected_devices/spec/lib/fitbit/fitbit_client_spec.rb:5:in `<top (required)>'

Comment (yonidoronpeters) [reply modules/dhp_connected_devices/spec/lib/fitbit_client_spec.rb:6]: Good catch! just pushed a fix

**Review by thilton-oddball — APPROVED** [review]

**Review by reblg — APPROVED** [review]
Looks awesome!

**Pull request merged**